orochi/client.py

```diff
@@ -685,6 +705,12 @@ def do_status(self, s=''):
             parts.append('(%s)' % track_year)
         print(' '.join(parts) + '.')
 
+        # Log the current song to a file (can be used by other programs)
+        if self._log_current_song is True:
+            f = open(expanduser("~") + '/.orochi.current_song.log', 'w')
```

Comment:
context manager is great..  not a python guy, so didn't know that :) will incorporate both.